packages/fxa-auth-db-mysql/lib/db/mem.js

```diff
@@ -1086,6 +1086,38 @@ module.exports = function (log, error) {
     }).reverse())
   }
 
+  Memory.prototype.securityEventsByUid = function (uid) {
+    if (! uid) {
+      return P.reject(error.notFound())
+    }
+    var key = uid.toString('hex')
+    var events = securityEvents[key] || []
+    var securityEventsLimit = 50
+
+    function filterEvents (events, eventsLimit, key) {
+      var filteredEvents = []
+
+      for (var i = 0; i < events.length; i++) {
```

Comment:
@shane-tomlinson I've used here vanilla for loop and not filter method with the index concept because of this reason

```js
// Suppose if events is a list of numbers and I want 2 events which are greater than 4, 
// then this index concept will only return me an empty array though [5, 6] should be returned
// and there's no method that can stop filtering events when it has 50 reached events 
// that's why I made this custom function.

var events = [1, 2, 3, 4, 5, 6, 7, 8]
var eventsLimit = 2
events.filter((ev, index) => {
    if(index < eventsLimit)
        return ev > 4
})
```

Reply:
That's fine, nit on the `var`, use `let` or `const` instead.